packages/ignite-core/tasks/check-node.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+const chalk = require("chalk");
+const errorHandler = require("../lib/error-handler");
+const logger = require("../lib/logger");
+const readline = require("readline");
+const xsh = require("xsh");
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+  terminal: false
+});
+
+const checkNode = function(type, igniteCore) {
+  return new Promise((resolve, reject) => {
+    return xsh
+      .exec(true, "node -v")
+      .then(function(nodeVersion) {
+        nodeVersion = nodeVersion.stdout.slice(0, -1);
+
+        return xsh
+          .exec(true, "npm -v")
+          .then(function(npmVersion) {
+            const checkNodePath = process.platform.startsWith("win")
+              ? "where node"
+              : "which node";
+            npmVersion = npmVersion.stdout.slice(0, -1);
+
+            return xsh
+              .exec(true, checkNodePath)
+              .then(function(nodePath) {
+                nodePath = nodePath.stdout.slice(0, -1);
+
+                logger.log(chalk.green(`Your Node version is: ${nodeVersion}`));
+                logger.log(chalk.green(`Your npm version is: ${npmVersion}`));
+                logger.log(
+                  chalk.green(`Your Node binary path is: ${nodePath}`)
+                );
```

Comment:
should do something like checking if `node` version is:

- less than 6 - print message: `"Recommend use Node LTS version 6"`
- `>= 6` - print message: `"You are using Node version ${...}.  It should work."`

npm version:

-  less than 3 - print message: `"Require npm version 3 and up"`
- `>= 3` - print message: `"Your npm version is ${...}.  It should work."`